public/graphql/get_component.graphql

```diff
@@ -0,0 +1,10 @@
+query Components ($component: String) {
```

Comment:
how about following a bit more REST-ish naming convention applied in other projects and calling the file `components/search` instead of `get_component`?

Reply:
The aim of this code to get only one component to component's page at the styleguide.
I'm waiting for Kalman to discuss thedetails of the solution.
(This is only a workaround at ths moment to serve the data to the frontend.)

Reply:
yeah I see, was looking at `{% function components_query_result = 'modules/components/lib/queries/get_components' %}` in the page, which fetches collection and wasn't precise with the comment - in this scenario, I meant  `components/find` instead of `components/search`, the point was about having the directory with the resource name

Reply:
I moved the files into `components` subfolder.